### PR TITLE
Add .sln file to F# interoperability sample

### PR DESCRIPTION
**Pull request by tcNickolas**
As @geduardo pointed out in #328, the instructions for the sample mention .sln file which does not exist.

### CODE REVIEW

**Comment by geduardo** [comment]
There is a build error with the `Quantum_Fsharp.sln` file. Apparently in Linux
##[error]Failed to build /home/vsts/work/1/s/samples/interoperability/fsharp/Quantum_Fsharp.sln.

**Comment by tcNickolas** [comment]
I'm not sure what happened there - it could be something like case sensitivity or direction of slashes in the path, but the paths seem similar to other .sln files in the repo which build fine. @cgranade any idea?

File: samples/interoperability/fsharp/Quantum_Fsharp.sln [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29905.134
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "QuantumCode", "QuantumCode\QuantumCode.csproj", "{2BD1572A-0892-43D4-9D82-115AD86A8378}"
+EndProject
+Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "FsharpDriver", "FsharpDriver\FsharpDriver.fsproj", "{6C0AC1D5-9C57-43DD-BC0B-3306A1B4CDF0}"
```
Comment (cgranade) [inline samples/interoperability/fsharp/Quantum_Fsharp.sln:8]:
```suggestion
Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "FSharpDriver", "FSharpDriver\FSharpDriver.fsproj", "{6C0AC1D5-9C57-43DD-BC0B-3306A1B4CDF0}"
```

**Comment by cgranade** [comment]
As you note,  @tcNickolas, since Linux is case sensitive, path names need to match capitalization as well. It looks like the problem may have been `Fsharp` versus `FSharp` in this case.

**Comment by tcNickolas** [comment]
Yes, that was it - upper case S vs lower case s is hard :-)

**Pull request merged**